src/common/constants/runtime.js

```diff
@@ -69,3 +71,5 @@ export const ffVersion = (() => {
 export const isIE = Boolean(isBrowserScope && window.document.documentMode) // deprecated property that only works in IE
 
 export const supportsSendBeacon = !!navigator.sendBeacon
+
+export const offset = Math.floor(globalScope?.performance?.timeOrigin || globalScope?.performance?.timing?.navigationStart || Date.now())
```

Comment:
Do you want to just change the runtime obj to use this too? This is being duplicated rn

Reply:
yes, because it is nice to be able to access without an identifier.  But offset is used in many places so we should do this separately.  Adding a ticket

Reply:
https://issues.newrelic.com/browse/NR-151155